composeApp/build.gradle.kts

```diff
@@ -0,0 +1,101 @@
+import com.android.build.gradle.internal.cxx.configure.gradleLocalProperties
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+plugins {
+    alias(libs.plugins.kotlin.multiplatform)
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.compose.multiplatform)
+    alias(libs.plugins.compose.compiler)
+    alias(libs.plugins.kotlin.parcelize)
+}
+
+kotlin {
+    androidTarget {
+        compilerOptions {
+            jvmTarget.set(JvmTarget.JVM_11)
+        }
+    }
+
+    sourceSets {
+        androidMain.dependencies {
+            implementation(project(":shared"))
```

Comment:
Could use type-safe accessors instead of a string

https://docs.gradle.org/current/userguide/declaring_dependencies_basics.html#sec:type-safe-project-accessors